8.0/Dockerfile

```diff
@@ -2,11 +2,11 @@
 
 # Do not edit individual Dockerfiles manually. Instead, please make changes to the Dockerfile.template, which will be used by the build script to generate Dockerfiles.
 
-FROM cimg/base:2022.01
+FROM cimg/base:2022.08
```

Comment:
i would change this to the newest version; this is almost a year old. otherwise, LGTM

Reply:
Done :+1: